From 5517661168dcce0a8fdfa51c7aa331853aebdf22 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 1 Jul 2015 23:01:42 -0500
Subject: [PATCH 5/5] Revert "of: make unittest select OF_EARLY_FLATTREE
 instead of depend on it"

This reverts commit 649e0a77e28a7796bf62bfda0fe3f2aee094bd58.
---
 drivers/of/Kconfig | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/of/Kconfig b/drivers/of/Kconfig
index 63ca0af..07bb3c8 100644
--- a/drivers/of/Kconfig
+++ b/drivers/of/Kconfig
@@ -9,8 +9,7 @@ menu "Device Tree and Open Firmware support"
 
 config OF_UNITTEST
 	bool "Device Tree runtime unit tests"
-	depends on OF_IRQ
-	select OF_EARLY_FLATTREE
+	depends on OF_IRQ && OF_EARLY_FLATTREE
 	select OF_RESOLVE
 	help
 	  This option builds in test cases for the device tree infrastructure
-- 
2.1.4

